packages/subgraph/manifest/data/rinkeby.json

```diff
@@ -29,6 +32,9 @@
   "votingAppIds": [
     { 
       "id": "0x39aa9e500efe56efda203714d12c78959ecbf71223162614ab5b56eaba014145", "name": "disputable-voting.precedence-campaign.aragonpm.eth"
+    },
+    { 
+      "id": "0x705b5084c67966bb8e4640b28bab7a1e51e03d209d84e3a04d2a4f7415f93b34", "name": "disputable-voting.aragonpm.eth"
```

Comment:
The id is for the `disputable-voting.open.aragonpm.eth` I think we name it wrong.